mat64/cholesky.go

```diff
@@ -81,7 +81,7 @@ func (t *Triangular) Cholesky(a *SymDense, upper bool) (ok bool) {
 // SolveCholesky finds the matrix x that solves A * X = B where A = L * L^T or
 // A = U^T * U, and U or L are represented by t. The matrix A must be symmetric
 // and positive definite.
-func (m *Dense) SolveCholesky(t *Triangular, b Matrix) {
+func (m *Dense) SolveCholesky(t *TriDense, b Matrix) {
```

Comment:
If you'd like it can, but the thought was that it would always be used after calling TriDense.Cholesky.